### PR TITLE
Reset textarea height when press submit key

### PR DESCRIPTION
**Pull request by ykzts**
ref #2236

### CODE REVIEW

File: app/assets/javascripts/components/features/compose/components/compose_form.jsx [hunk 1]
```diff
@@ -44,12 +44,13 @@ class ComposeForm extends React.PureComponent {
 
   handleKeyDown (e) {
     if (e.keyCode === 13 && (e.ctrlKey || e.metaKey)) {
+      this.autosuggestTextarea.reset();
       this.props.onSubmit();
```
Comment (stephenburgess8) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: You could call `handleSubmit` here for a little DRYer code.

Comment (ykzts) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: Is it like this? 777336d

**Review by Gargron — APPROVED** [review]

File: app/assets/javascripts/components/features/compose/components/compose_form.jsx [hunk 2]
```diff
@@ -44,12 +44,12 @@ class ComposeForm extends React.PureComponent {
 
   handleKeyDown (e) {
     if (e.keyCode === 13 && (e.ctrlKey || e.metaKey)) {
-      this.props.onSubmit();
+      this.props.handleSubmit();
```
Comment (Gargron) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: this isn't a prop! just this.handleSubmit()

**Review by Gargron — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/components/features/compose/components/compose_form.jsx (hunk 2 above)
Comment (ykzts) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: I seem to be tired.

Comment (ykzts) [inline app/assets/javascripts/components/features/compose/components/compose_form.jsx]: 974ab5f I'm sorry. I fixed it...

**Review by Gargron — APPROVED** [review]

**Pull request merged**